addOns/spider/src/main/java/org/zaproxy/addon/spider/parser/SpiderHttpHeaderParser.java

```diff
@@ -51,9 +52,12 @@ public boolean parseResource(ParseContext ctx) {
         }
 
         // Link header - potentially multiple absolute or relative URLs in < >
-        // TODO replace when next core version available HttpHeader.LINK
-        String link = message.getResponseHeader().getHeader("Link");
-        if (link != null && !link.isEmpty()) {
+        List<String> links =
+                message.getResponseHeader().getHeaderValues(HttpFieldsNames.LINK).stream()
+                        .filter(headerValue -> headerValue != null && !headerValue.isEmpty())
+                        .collect(Collectors.toList());
```

Comment:
It would be better to process them immediately with `forEach` rather than collect, or skip them in the for.